Assets/Mirror/Editor/Weaver/CompilationFinishedHook.cs

```diff
@@ -46,32 +49,29 @@ static void HandleError(string msg)
         [InitializeOnLoadMethod]
         static void OnInitializeOnLoad()
         {
-            CompilationPipeline.assemblyCompilationFinished += OnCompilationFinished;
```

Comment:
why is this not needed anymore?

Reply:
it's still there, just moved down...and commented